rs/nns/integration_tests/BUILD.bazel

```diff
@@ -424,3 +424,47 @@ rust_ic_test(
     ],
     deps = DEPENDENCIES + DEV_DEPENDENCIES,
 )
+
+# Adopts a given NNS proposal in a test network with the golden state, asserts successful execution.
+#
+# Run this test as follows:
+#
+#     bazel \
+#         test \
+#         --test_env=SSH_AUTH_SOCK \
+#         --test_env=NNS_PROPOSAL_ID_TO_ADOPT=134803 \
```

Comment:
Sounds like this test will break once that proposal is passed. I guess the idea is that we will delete this when that happens?

Reply:
Or, I guess we can hang onto this, so that if the same question comes up again for another proposal in the future, we just need to slightly modify this command. Maybe add a comment that explains that this is not run regularly, but kept around for occasions of this type.

Reply:
Closing this as I think it's helpful to provide a concrete example in the docs.